### PR TITLE
tests(playwright): enable back playwright tests

### PR DESCRIPTION
**Pull request by louis-bompart**
+add eslint rule for promises on playwright specs

+touch up on some tests

https://coveord.atlassian.net/browse/KIT-3459

### CODE REVIEW

File: .github/workflows/prbot.yml [hunk 1]
```diff
@@ -117,7 +117,7 @@ jobs:
           app-id: ${{ secrets.GH_APP_ID }}
           private-key: ${{ secrets.GH_APP_PRIVATE_KEY }}
           owner: coveo
-          repo: ui-kit-prs
```
Comment (louis-bompart) [inline .github/workflows/prbot.yml:120]: https://github.com/actions/create-github-app-token/tree/31c86eb3b33c9b601a1f60f98dcbfd1d70f379b4/?tab=readme-ov-file#create-a-token-for-multiple-repositories-in-the-current-owners-installation

File: packages/atomic/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    {
+      files: '**/*.e2e.ts',
+      parserOptions: {
+        project: './tsconfig.playwright.json',
+        tsconfigRootDir: __dirname,
+      },
+      rules: {
+        '@typescript-eslint/no-floating-promises': 'error',
+      },
+    },
```
Comment (louis-bompart) [inline packages/atomic/.eslintrc.js:36]: That's what I "really" added.
The rest is from the .json previous file

File: packages/atomic/src/components/search/facets/atomic-timeframe-facet/e2e/atomic-timeframe-facet.e2e.ts
Comment (louis-bompart) [inline packages/atomic/src/components/search/facets/atomic-timeframe-facet/e2e/atomic-timeframe-facet.e2e.ts:1]: don't need to click before fill ;)

File: packages/atomic/src/components/search/facets/atomic-timeframe-facet/e2e/atomic-timeframe-facet.e2e.ts [hunk 3]
```diff
@@ -126,16 +120,13 @@ test.describe('with min and max values', () => {
...
-      await facet.facetInputStart.click();
       await facet.facetInputStart.fill('2021-01-01');
-      await facet.facetInputEnd.click();
       await facet.facetInputEnd.fill('2021-01-31');
     });
 
     test.describe('when clicking the apply button', () => {
       test.beforeEach(async ({facet}) => {
         await facet.facetApplyButton.click();
-        await facet.facetClearFilter.waitFor({state: 'visible'});
```
Comment (louis-bompart) [inline packages/atomic/src/components/search/facets/atomic-timeframe-facet/e2e/atomic-timeframe-facet.e2e.ts:138]: Not really required done in the following beforeEach

File: packages/atomic/src/components/search/tabs/atomic-tab-manager/e2e/atomic-tab-manager.e2e.ts [hunk 4]
```diff
@@ -1,196 +1,198 @@
...
-test.beforeEach(async ({tabManager}) => {
-  await tabManager.load();
-  await tabManager.hydrated.waitFor();
-});
-
-test.describe('when viewport is large enough to display all tabs', () => {
-  test('should be A11y compliant', async ({makeAxeBuilder}) => {
-    const accessibilityResults = await makeAxeBuilder().analyze();
-    expect(accessibilityResults.violations).toEqual([]);
+test.describe('AtomicTabManager', () => {
```
Comment (louis-bompart) [inline packages/atomic/src/components/search/tabs/atomic-tab-manager/e2e/atomic-tab-manager.e2e.ts:3]: It's preferable to have a 'top-level' describe.

File: packages/atomic/src/components/search/tabs/atomic-tab-manager/e2e/atomic-tab-manager.e2e.ts
Comment (louis-bompart) [inline packages/atomic/src/components/search/tabs/atomic-tab-manager/e2e/atomic-tab-manager.e2e.ts:1]: I just added the top-level describe 😅

**Review by alexprudhomme — COMMENTED** [review]
~~Looks good but seems like query summary tests are still failing + there are eslint errors in cypress to handle.~~

Nevermind, tests are passing. Only a github front end bug !

File: packages/atomic/src/components/search/tabs/atomic-tab-manager/e2e/atomic-tab-manager.e2e.ts (hunk 4 above)
Comment (alexprudhomme) [reply packages/atomic/src/components/search/tabs/atomic-tab-manager/e2e/atomic-tab-manager.e2e.ts:3]: Agree !

**Review by alexprudhomme — APPROVED** [review]
Looks good but there are now eslint errors in cypress files !

File: .github/workflows/prbot.yml (hunk 1 above)
Comment (fbeaudoincoveo) [reply .github/workflows/prbot.yml:120]: I wonder why they didn't make this an array...

**Review by fbeaudoincoveo — APPROVED** [review]

File: .github/workflows/prbot.yml (hunk 1 above)
Comment (louis-bompart) [reply .github/workflows/prbot.yml:120]: Arrays is special in yml let's say, and hard to interpret.
Often you'll see toJson and the like in GHA to go thru the hoop and cast that to string or other primitive (string, number, boolean)

**Pull request merged**